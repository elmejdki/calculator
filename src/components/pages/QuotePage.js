import React from 'react';

const QuotePage = () => (
  <div>
    <h1>Quote Page</h1>
  </div>
);

export default QuotePage;
