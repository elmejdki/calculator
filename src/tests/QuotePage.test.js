import React from 'react';
import renderer from 'react-test-renderer';
import QuotePage from '../components/pages/QuotePage';

it('renders correctly the Quote Page', () => {
  const tree = renderer.create(<QuotePage />).toJSON();

  expect(tree).toMatchSnapshot();
});
