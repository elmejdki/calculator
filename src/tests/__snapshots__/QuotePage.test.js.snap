// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly the Quote Page 1`] = `
<div
  className="max-height max-width quote"
>
  <h1>
    Quote Of The Day:
  </h1>
  <div>
    <p>
      <strong>
        Mathematics 
      </strong>
      is not about numbers, equations, computations or algorithms it is about
      <strong>
         Understanding.
      </strong>
      <span>
         ~ Wiliam Paul Thurston
      </span>
    </p>
  </div>
</div>
`;
